### ISSUE TITLE
Log4J error when trying to start application

### ISSUE THREAD

**Issue by Phoenix616** [tectonicus/tectonicus#197]
When trying to run a render with an application built from the latest code base (with `./gradlew build`) the following errors and stack trace is shown and it fails to do anything: 

```
ERROR StatusLogger Log4j2 could not find a logging implementation. Please add log4j-core to the classpath. Using SimpleLogger to log to the console...
ERROR StatusLogger LogManager returned an instance of org.apache.logging.log4j.simple.SimpleLoggerContextFactory which does not implement org.apache.logging.log4j.core.impl.Log4jContextFactory. Unable to initialize Log4j.
WARNING: sun.reflect.Reflection.getCallerClass is not supported. This will impact performance.
Exception in thread "main" java.lang.ClassCastException: class org.apache.logging.log4j.simple.SimpleLoggerContext cannot be cast to class org.apache.logging.log4j.core.LoggerContext (org.apache.logging.log4j.simple.SimpleLoggerContext and org.apache.logging.log4j.core.LoggerContext are in unnamed module of loader 'app')
        at org.apache.logging.log4j.core.LoggerContext.getContext(LoggerContext.java:239)
        at org.apache.logging.log4j.core.config.Configurator.setRootLevel(Configurator.java:439)
        at tectonicus.TectonicusApp.<init>(TectonicusApp.java:62)
        at tectonicus.TectonicusApp.main(TectonicusApp.java:292)
(base)
```

Removing the `Configurator.setRootLevel(config.getLoggingLevel());` makes it start (but the first error is still shown) and of course the log level will be wrong.

Using `OpenJDK Runtime Environment Temurin-17.0.3+7 (build 17.0.3+7)`

**Comment by skoeven** [comment]
Are you trying to use the Tectonicus jar by itself?  It needs the log4j jars to be in the same directory. It's setup like that for you in the zip file that gets created in the distributions folder after running the build.

**Comment by Phoenix616** [comment]
Oh, yeah I tried to run the jar directly as I'm used to it including all required dependencies. You might want to add some documentation that all the files are required. (as well as for building) Seeing as I built it myself directly as I wanted the latest changes I never actually took a look at what the release zip file contained so just assumed it would work like everything else by taking the output jar. ^^'
